### PR TITLE
Add follow options to follow symbolink links

### LINKED ISSUE #40: copyfiles doesn't follow directories that are symlinks
As in title
This is consistent with `glob` which by default doesn't follow folder symlink. Howewver, `glob` supports `follow` flag to override this. It would be cool is `copyfiles` support that too.

Adding this when preparing `globOpts` helps:
```
if (config.follow) {
    globOpts.follow = true;
}
```

I will prepare PR for this if there is positive feedback.

### PR DESCRIPTION
**Pull request by axelhzf**
Fix #40

### CODE REVIEW

File: copyfiles [hunk 1]
```diff
@@ -26,9 +26,12 @@ args.alias('a', 'all')
   .alias('s','soft')
   .boolean('s')
   .describe('s', 'do not overwrite destination files if they exist')
-  .usage('$0 [options] inFile [more files ...] outDirectory').help()
+  .boolean('follow')
```
Comment (calvinmetcalf) [inline copyfiles]: can we get a short alias for this?

Comment (axelhzf) [reply copyfiles]: sure, but `f` was already taken by `flatten`. Any suggestion?

Comment (calvinmetcalf) [reply copyfiles]: upper case F ?

Comment (axelhzf) [reply copyfiles]: alias added :D

**Comment by calvinmetcalf** [comment]
v2.2.0

**Pull request merged**